2_process/src/process_precip.R

```diff
@@ -141,7 +141,7 @@ process_precip_rasters <- function(ind_file, precip_spatial_ind,
   precip_spatial <- readRDS(sc_retrieve(precip_spatial_ind)) %>%
     dplyr::select(-x, -y)
 
-  time <- dplyr::select(precip, time_chr) %>%
+  time_df <- dplyr::select(precip, time_chr) %>%
```

Comment:
weird, was this a bug before? I don't see how it would've worked w/o this edit

Reply:
It was a part I forgot to copy from Laura. It failed before it actually tried to use `time_df` because of the data only having one timestep this time around, so I only noticed it once we got past that